### DISCUSSION TITLE
How to use custom class as Option Type in AutoComplete.

### DISCUSSION THREAD

**Discussion by patelyahya** [ant-design-blazor/ant-design-blazor#3445] [Q&A]
Hi, this is Yahya Patel. I wonder how would I use Autocomplete feature with my custom class? I couldn't find any documentation for custom option type.

**Comment by ElderJames** [comment]
Just use `Class` and `Style` parameters, they are for all components.

**Reply by patelyahya** [reply]
Sorry, I couldn't make it clear. Actually I am talking about C# class object. By Default, `AutoCompleteOption` is the class type of Selected Item. I want to replace `AutoCompleteOption` with my own type.
